environment/src/main/resources/application.yml

```diff
@@ -6,6 +6,10 @@ opentracing:
   jaeger:
     log-spans: false
 
+audit:
+  service:
+    enabled: false
+
```

Comment:
remove please

Reply:
we need to add this variable because the structure event module is used by all module and we set the default value in application.yml.